i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-servers-hp-ilo-xmlapi.md

```diff
@@ -2,34 +2,295 @@
 id: hardware-servers-hp-ilo-xmlapi
 title: HP Ilo XMLAPI
 ---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
 
-## Prerequisites
+## Contenu du pack
 
-### Centreon Plugin
+### Modèles
 
-Install this plugin on each needed poller:
+Le connecteur de supervision **HP Ilo XMLAPI** apporte un modèle d'hôte :
 
-``` shell
+* **HW-Server-Hp-Ilo-Xmlapi-custom**
+
+Le connecteur apporte le modèle de service suivant
+(classé selon le modèle d'hôte auquel il est rattaché) :
+
+<Tabs groupId="sync">
+<TabItem value="HW-Server-Hp-Ilo-Xmlapi-custom" label="HW-Server-Hp-Ilo-Xmlapi-custom">
+
+| Alias           | Modèle de service                       | Description             |
+|:----------------|:----------------------------------------|:------------------------|
+| Hardware-Global | HW-Hp-Ilo-Hardware-Global-Xmlapi-custom | Contrôle les composants |
+
+> Les services listés ci-dessus sont créés automatiquement lorsque le modèle d'hôte **HW-Server-Hp-Ilo-Xmlapi-custom** est utilisé.
+
+</TabItem>
+</Tabs>
+
+### Métriques & statuts collectés
+
+Voici le tableau des services pour ce connecteur, détaillant les métriques rattachées à chaque service.
+
+<Tabs groupId="sync">
+<TabItem value="Hardware-Global" label="Hardware-Global">
+
+La liste des métriques varie d'un modèle à l'autre.
+
+</TabItem>
+</Tabs>
+
+## Prérequis
+
```

Comment:
what about this sentence in the old doc? "A reading account is needed on HP ILO card."